ui/shared/Clipboard.jsx

```diff
@@ -1,26 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faClipboard } from '@fortawesome/free-regular-svg-icons';
+import {
+  faClipboard,
+  faCheckCircle,
+} from '@fortawesome/free-regular-svg-icons';
 import { Button } from 'reactstrap';
 
-const Clipboard = ({ description, text, outline, visible, color }) => {
+const Clipboard = ({ description, text, outline, color }) => {
+  const [copied, setCopied] = useState(false);
```

Comment:
I see that we are using `hooks` here. At the moment they are not used anywhere else in the project. We should discuss with the team if it is suitable to integrate them in the future.
To use `state` we should change from `functional component` to `class component`.